packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -213,6 +213,35 @@ abstract class WidgetsBindingObserver {
   ///    boilerplate.
   void didChangeTextScaleFactor() { }
 
+  /// Called when the platform brightness changes.
```

Comment:
Looks like this comment is the same as above? Maybe use an `{@template}`.

Reply:
Done